src/Linter/LintingException.php

```diff
@@ -16,6 +16,8 @@
 
 /**
  * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * @final @TODO make class "final" on next major 4.0
```

Comment:
```suggestion
 * @final
 * @TODO 4.0 make class "final"
```

Reply:
just for sake of easier finding `@TODO 4.0` (like i did for 3.0 ;) )
but not a blocker

feel free to reject or squash to your commit